.pipelines/stages/py-publishing-stage.yml

```diff
@@ -0,0 +1,117 @@
+parameters:
+- name: SpecificArtifact
+  displayName: Use Specific Artifact
+  type: boolean
+  default: false
+- name: BuildId
+  displayName: Specific Artifact's BuildId
+  type: string
+  default: '0'
+- name: enable_win_cpu
+  type: boolean
+- name: enable_win_cuda
+  type: boolean
+- name: enable_win_dml
+  type: boolean
+- name: enable_win_arm64_cpu
+  type: boolean
+- name: enable_linux_cpu
+  type: boolean
+- name: enable_linux_cuda
+  type: boolean
+- name: enable_linux_rocm
+  type: boolean
+- name: enable_macos_cpu
+  type: boolean
+- name: cuda_versions
+  type: string
+  default: '12.2'
+
+stages:
+- stage: python_publishing
+  jobs:
+  - ${{ if eq(parameters.enable_win_cpu, true) }}:
```

Comment:
You may put each job in a separate stage, then you can remove all the "enable_xxxx" parameters here.  You can choose the stages in ADO's UI.